scss/modules/_nav-bar.scss

```diff
@@ -1,62 +1,70 @@
 // Nav Bar View
 .nav-bar-container {
-	background-color: $nav-bar-green;
+	font-family: 'Work Sans', sans-serif;
+	background:
+	linear-gradient(
+		rgba(32, 90, 66, 0.45),
+		rgba(32, 90, 66, 0.45)
+	),
+	url(http://wallpaperlab.net/wp-content/uploads/2015/09/Black-and-white-mountains-in-fog.jpg);
+
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-position: center;
 	color: white;
 	float: left;
 	height: 100vh;
 	max-width: 13em;
 	position: relative;
 	width: 100%;
-}
 
-.nav-bar-user {
-  background: rgba(0,0,0,.1);
-  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
   margin: 0 auto;
   padding: 20px 0;
   text-align: center;
 
 	p {
 		font-size: 18px;
-		margin-bottom: 7px;
+		margin-bottom: 10px;
 
 		&:last-child {
 			font-size: 14px;
-			margin-bottom: 0;
+			margin-bottom: 10px;
 		}
 	}
 }
-
 .nav-bar-avatar {
 	border-radius: 50%;
 	margin-bottom: 10px;
 	width: 60px;
 }
 
-.nav-bar-links {
-	padding: 0;
+.nav-bar {
+  width:100%;
+  height:1px;
+  border-bottom: 1px solid rgba(255,255,255,.25);
+}
 
-	ul {
+ul {
```

Comment:
@sarjford  From how I'm reading this, the 'ul' is not nested within any classes, so you're targeting ALL 'ul' elements that exist. This could be a problem if other views have a 'ul' that is being used. It wouldn't hurt to be nested at least one level.